db/schema/005_vehicle_updates.sql

```diff
@@ -0,0 +1,44 @@
+-- +goose Up
+
+ALTER TABLE vehicle ADD gtfs_hash CHARACTER VARYING NOT NULL DEFAULT '';
+ALTER TABLE vehicle ALTER COLUMN gtfs_hash DROP DEFAULT;
+
```

Comment:
The `gtfs_hash` column was added as a performance optimization for trips and alerts. I don't expect it's going to be much of an optimization for vehicles. The reason why trip and alert updates are slowish is because each one touches many rows (for trips you have to update all of the trip stop times, for alerts you have update all of the informed entity tables). Whereas vehicle updates really just touch the vehicle row itself. However if you think the code complexity of this `gtfs_hash` system is not too bad, we can just keep it anyway.

Reply:
I see, yeah I think this was premature optimization on my part. I'll remove for now to simplify things and should be easy to re-add if needed in the future.